pkg/client/didexchange/client.go

```diff
@@ -42,7 +42,7 @@ type Client struct {
 	service.Action
 	service.Message
 	didexchangeSvc           service.DIDComm
-	wallet                   wallet.Crypto
+	kms                      kms.Crypto
```

Comment:
Note: Crypto should be a sibling to kms.

Reply:
I will refactor this further...

Reply:
renamed to `KeyManager`